pandas/core/indexes/base.py

```diff
@@ -4017,8 +4017,52 @@ def unique(self, level=None):
         result = super(Index, self).unique()
         return self._shallow_copy(result)
 
-    @Appender(base._shared_docs['drop_duplicates'] % _index_doc_kwargs)
     def drop_duplicates(self, keep='first'):
+        """
+        Return Index with duplicate values removed.
+
+        The drop_duplicates method can remove occurences or whole sets
```

Comment:
I'm not sure what "whole sets" means here. Personally I think this extended summary can be removed. The first line is descriptive enough.

Reply:
Ok, thanks Tom, I'll adjust accordingly!